conformance/tests/callables_kwargs.py

```diff
@@ -121,3 +121,19 @@ def func5(v1: int, **kwargs: Unpack[TD2]) -> None:  # E: parameter v1 overlaps w
 
 def func6(**kwargs: Unpack[T]) -> None:  # E: unpacked value must be a TypedDict, not a TypeVar bound to TypedDict.
     ...
+
+
+def func7(*, v1: int, v3: str, v2: str = "") -> None:
```

Comment:
Please add the quoted text from the spec for each of the test cases. I see that you added it for the second test case but not the first. In both cases, use the `# >` quoting syntax so it's consistent with all of the other tests and the [README guidance](https://github.com/python/typing/blob/main/conformance/README.md#notes-about-tests).